testing/tests/DevExpress.ui.widgets/treeViewParts/optionChanged.js

```diff
@@ -88,6 +88,76 @@ QUnit.test("searchValue from value to empty", function(assert) {
     assert.equal($items.length, 6, "6 items were rendered after filtration");
 });
 
+QUnit.test("search should consider dataSource simple sorting", function(assert) {
+    var data = [
+            { id: 1, parentId: 0, text: "Bikes" },
+            { id: 4, parentId: 3, text: "BMW" },
+            { id: 13, parentId: 3, text: "Audi" },
+            { id: 3, parentId: 0, text: "Cars" },
+            { id: 11, parentId: 10, text: "YX 1" },
+            { id: 12, parentId: 10, text: "YX 2" },
+            { id: 14, parentId: 13, text: "A1" },
+            { id: 15, parentId: 13, text: "A5" },
+            { id: 2, parentId: 0, text: "Motobikes" },
+            { id: 5, parentId: 4, text: "X1" },
+            { id: 6, parentId: 4, text: "X5" },
+            { id: 7, parentId: 4, text: "X6" },
+            { id: 10, parentId: 2, text: "Yamaha" },
+            { id: 8, parentId: 1, text: "Stels" },
+            { id: 9, parentId: 2, text: "Honda" }
+        ],
+        treeView = initTree({
+            dataSource: { store: data, sort: "text" },
+            dataStructure: "plain",
+            parentIdExpr: "parentId",
+            keyExpr: "id"
+        }).dxTreeView("instance");
+
+    treeView.option("searchValue", "1");
+
+    var $items = $(treeView.$element()).find(".dx-treeview-item"),
+        expectedValues = ["Cars", "Audi", "A1", "BMW", "X1", "Motobikes", "Yamaha", "YX 1"];
+
+    $.each($items, function(index, item) {
+        assert.equal($(item).text(), expectedValues[index], "Correct item");
+    });
+});
+
+QUnit.test("search should consider dataSource sorting expression", function(assert) {
```

Comment:
May be this test is extra, because it tests the store method?...

Reply:
Nope. It tests that the data adapter receive a sorting expression and correctly handle it. I think it necessary to test this case, but probably better move it to the data adapter tests.

Reply:
Okey, sounds greater)